src/raftstore/store/local_metrics.rs

```diff
@@ -23,6 +23,9 @@ pub struct RaftReadyMetrics {
     pub commit: u64,
     pub append: u64,
     pub snapshot: u64,
+    pub read_state: u64,
```

Comment:
read_state is for raft ReadIndex count, soft/hard state is for raft state changed.